css/style.css

```diff
@@ -32,3 +39,6 @@ input {
   pointer-events: none;
   opacity: 0.5;
 }
+#Load-more, #Show-less{
```

Comment:
please use one uniform naming convention. 

```suggestion
#load-more, #show-less{
```

Reply:
done!